base_report_to_printer/printing.py

```diff
@@ -127,10 +128,23 @@ def print_document(self, report, content, format):
             os.write(fd, content)
         finally:
             os.close(fd)
-        connection = cups.Connection()
+
+        try:
+            _logger.debug(
+                'Starting to connect to CUPS on %s:%s'
+                % (CUPS_HOST, CUPS_PORT))
+            connection = cups.Connection(CUPS_HOST, CUPS_PORT)
+            _logger.debug('Connection to CUPS successfull')
+        except:
+            raise Warning(
+                _("Failed to connect to the CUPS server on %s:%s. "
+                    "Check that the CUPS server is running and that "
+                    "you can reach it from the Odoo server.")
+                % (CUPS_HOST, CUPS_PORT))
 
         options = self.print_options(report, format)
 
+        _logger.debug('Sending job to CUPS printer %s' % self.system_name)
```

Comment:
Nitpicking: maybe it's worth mentioning the values of CUPS_HOST and CUPS_PORT here, so that we don't go searching for jobs on the wrong host?